monodbassignment/superadmins.json

```diff
@@ -0,0 +1,10 @@
+{
+  "id":"String datatype", 
+  "name": "String datatype",
+  "passwd":"String datatype",
+  "email": "string datatype",
+   "contact_no": "number datatype",
+  "dob": "date datatype",
+  "editors":["contain all editors which are handle by this superadmin"],
```

Comment:
As a super admin who has access to all the users and posts, you don't need to store all the user ids in the db. 
Everytime you add or remove a user/post, you would have needed to update each super admin, which is unnecessary.